extensions/applicationinsights-analytics-js/src/JavaScriptSDK/AnalyticsPlugin.ts

```diff
@@ -53,55 +57,86 @@ function _getReason(error: any) {
     return error || "";
 }
 
-export class ApplicationInsights extends BaseTelemetryPlugin implements IAppInsights, IAppInsightsInternal {
```

Comment:
renamed to AnalyticsPlugin to reduce the number of issue about having multiple "ApplicationInsights" classes.

Reply:
do you have a list of all breaking changes?

Reply:
No, but this one is not actually a breaking change as I re-export as the original name as well.